docs/src/admin-guide/finalize-registration.md

```diff
@@ -0,0 +1,55 @@
+# How to finalize registration
+
+Who is this guide for: admin
+
+Users are free to register to makerspace.se, but they are only able to
```

Comment:
> to makerspace.se

should probably be "in the member pages of Makeradmin" / "in the Makeradmin web shop"?